### PR TITLE
Added flask server support for sudoku and other static pages

### LINKED ISSUE #43: API Endpoint - Fetch sudoku board
We need a script to run in the background on a server somewhere that generates valid Sudoku boards. Feel free to look into it yourself, but it's a hard problem. So we aren't going to try and solve it. Instead we should use one of the many Sudoku generators that are open-source

### PR DESCRIPTION
**Pull request by JustinGOrefice**
Closes #43

### CODE REVIEW

File: src/Views/index.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ export { Instructor } from "./Pages/Instructor";
 export { Profile } from "./Pages/Profile";
 
 export { Dictionary } from "./Tools/Dictionary";
-export { Flashcards } from "./Tools/Flashcards";
+export { Flashcards } from "./Tools/FlashCards";
```
Comment (JerameyATyler) [inline src/Views/index.js]: This should be lowercase. I've changed this at least 5 times now.

File: src/Views/Games/Sudoku/Sudoku.js [hunk 2]
```diff
@@ -128,7 +41,16 @@ function Sudoku() {
     };
 
     useEffect(() => {
-        setBoard(getBoard());
+        fetch("http://127.0.0.1:5000/sudoku?difficulty=medium", {
```
Comment (JerameyATyler) [inline src/Views/Games/Sudoku/Sudoku.js]: this needs to be the correct URL and difficulty should be a variable.

File: src/Views/Games/Sudoku/Sudoku.js [hunk 3]
```diff
@@ -33,93 +33,6 @@ function Sudoku() {
 
     const [board, setBoard] = useState([]);
```
Comment (JerameyATyler) [inline src/Views/Games/Sudoku/Sudoku.js:34]: We need the getBoard function... to fetch boards

File: src/Views/Games/Sudoku/Board.js [hunk 4]
```diff
@@ -44,10 +41,29 @@ function Board({ boardState }) {
...
     const [selectedTile, setSelectedTile] = useState(null);
-    // To check the board we need to check rows, columns, and squares. Returns true if the board is complete, false otherwise
-    function checkBoard() {}
+
+    function checkBoard() {
+        for (let i = 0; i < 9; i++) {
+            let row = getRow(i);
+            let column = getColumn(i);
+            let square = getSquare(i);
+            for (let j = 0; j < 9; j++) {
```
Comment (JerameyATyler) [inline src/Views/Games/Sudoku/Board.js]: This inner loop is unnecessary. Use sets and set operations instead. What would the union of all the rows, columns, and squares look like if it's complete vs incomplete

**Review by JerameyATyler — CHANGES_REQUESTED** [review]

File: src/Views/Games/Sudoku/Sudoku.js (hunk 2 above)
Comment (JustinGOrefice) [reply src/Views/Games/Sudoku/Sudoku.js]: How do I get the correct URL?

Comment (JerameyATyler) [reply src/Views/Games/Sudoku/Sudoku.js]: api.speakeasy.services/sudoku?difficulty=medium

**Pull request closed without merging**